src/azure-cli/azure/cli/command_modules/resource/_json_handler.py

```diff
@@ -0,0 +1,220 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
```

Comment:
Let's put the origin of the code here: https://github.com/tikitu/jsmin/blob/master/jsmin/__init__.py

Reply:
Done